libbeat/_meta/config.reference.yml

```diff
@@ -137,6 +137,14 @@
 #       equals:
 #           http.code: 200
 #
+# The following example renames the field a to b:
```

Comment:
@dedemorton The processor part in the reference file becomes too extensive in my opinion. We should find a way to link here to the docs instead. WDYT?

Reply:
@ruflin I agree with you. We need to be careful about linking to docs, tho. We need to make sure the links get validated. I can investigate. We probably shouldn't hold up the merge of this PR for that, tho.